python/sparkdl/graph/builder.py

```diff
@@ -75,6 +78,48 @@ def run(self, *args, **kargs):
         """
         return self.sess.run(*args, **kargs)
 
+    def asUDF(self, udf_name, fetches, feed_dict=None, blocked=False, register=True):
+        """ 
+        Make the TensorFlow graph as a SQL UDF 
+
+        :param udf_name: str, name of the SQL UDF 
+        :param fetches: list, output tensors of the graph
+        :param feed_dict: dict, a dictionary that maps graph elements to input values
+        :param blocked: bool, whether the TensorFrame execution should be blocked based or row based
+        :param register: bool, whether this UDF should be registered
+        :return: JVM function handle object
+        """ 
+        # pylint: disable=W0212
+        # TODO: work with registered expansions
+        jvm_builder = JVMAPI.forClass(JVMAPI.MODEL_FACTORY_CLASSNAME)
```

Comment:
In retrospect, I am not a fan for calling `forClass` because it also sets the default the sql context. Something like `createBuilder` is good, I think.